experiment-rollout.js

```diff
@@ -46,33 +46,53 @@ const data = {
         priority: [
             {
                 status: 2,
-                name: 'Brands'
+                name: 'Brands / Large bots | Reserved unless Discord partner for brands'
             },
             {
                 status: 2,
-                name: 'Discord Staff'
+                name: 'Discord Staff | Staff and personal accounts'
             },
             {
                 status: 2,
-                name: 'Partnered, subscription-enabled (100USD/mo+), and verified server owners (includes some old partners + some content creators)'
+                name: 'Partnered / verified / ex-partnered / server subscription server owners'
             },
             {
                 status: 1,
-                name: 'Verified bots (Server count wins name conflicts, other ones will get their discriminator appended)'
+                name: 'Verified bots | Ordered by guild count'
```

Comment:
```suggestion
                name: 'Verified bots | Ordered by server count'
```

"server" can be clearer